homeassistant/components/unifi/device_tracker.py

```diff
@@ -262,12 +264,15 @@ def device_state_attributes(self):
         """Return the client state attributes."""
         attributes = {}
 
-        for variable in DEVICE_ATTRIBUTES:
-            if variable in self.client.raw:
-                attributes[variable] = self.client.raw[variable]
-
         attributes["is_wired"] = self.is_wired
 
+        for variable in CLIENT_STATIC_ATTRIBUTES + CLIENT_CONNECTED_ATTRIBUTES:
+            if variable in self.client.raw:
+                if self.is_disconnected and variable in CLIENT_CONNECTED_ATTRIBUTES:
+                    attributes[variable] = None
```

Comment:
Our standard is to not include the item at all if the value is `None`.

Reply:
Users are responsible for checking the existence of attributes in automations. That's standard.

Reply:
Allright, I will update the PR to not set them instead

Reply:
Here are the docs for the helpers we have to check state attributes:
https://www.home-assistant.io/docs/configuration/templating/#states

Reply:
Fixed